distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalogValidation.java

```diff
@@ -94,115 +103,108 @@ public void beforeExam() throws Exception {
         }
     }
 
+    @Before
+    public void setup() throws Exception {
+        configureShowInvalidMetacards("true", "true", getAdminConfig());
+        configureFilterInvalidMetacards("false", "false", getAdminConfig());
+        configureEnforceValidityErrorsAndWarnings("true", "false", getAdminConfig());
+        configureMetacardValidityFilterPlugin(Arrays.asList(""), getAdminConfig());
+        console.runCommand("catalog:removeall -f");
+        console.runCommand("catalog:removeall -f --cache");
+        //verify catalog is clean
+        expect("Failed to clear catalog").within(10, TimeUnit.SECONDS)
```

Comment:
@emmberk was wondering the same thing about some of these methods. I was thinking most of the code was fairly specific to this test but in this case I could probably pull this out.